bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,276 @@
+        )
+        await ctx.send(embed=e)
+        self._remove_user(user.id)
+
+    @nomination_group.group(name='edit', aliases=('e',), invoke_without_command=True)
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def nomination_edit_group(self, ctx: Context) -> None:
+        """Highlights the activity of helper nominees by relaying their messages to TalentPool."""
+
+        await ctx.invoke(self.bot.get_command("help"), "talentpool", "edit")
+
+    @nomination_edit_group.command(name='reason')
+    @with_role(Roles.owner, Roles.admin, Roles.moderator)
+    async def edit_reason_command(self, ctx: Context, nomination_id: int, *, reason: str) -> None:
+        """
+        Edits the reason/unnominate reason for the nomination with the given `id` depending on the status.
+
+        If the nomination is active, the reason for nominating the user will be edited;
+        If the nomination is no longer active, the reason for ending the nomination will be edited instead.
+        """
+        try:
+            nomination = await self.bot.api_client.get(f"{self.api_endpoint}/{nomination_id}")
+        except ClientResponseError as e:
+            if e.status == 404:
+                self.log.trace(f"Nomination API 404: Can't nomination with id {nomination_id}")
+                e = Embed(
+                    description=f":x: **Can't find a nomination with id `{nomination_id}`**",
+                    color=Color.red()
+                )
+                await ctx.send(embed=e)
+                return
+            else:
+                raise
+
+        field = "reason" if nomination["active"] else "end_reason"
+
+        self.log.trace(f"Changing {field} for nomination with id {nomination_id} to {reason}")
+        await self.bot.api_client.patch(
+            f"{self.api_endpoint}/{nomination_id}",
+            json={field: reason}
+        )
+
+        e = Embed(
+            description=f":white_check_mark: **Updated the {field} of the nomination!**",
+            color=Color.green()
+        )
+        await ctx.send(embed=e)
+
+    def _nomination_to_string(self, nomination_object: dict) -> str:
+        """Creates a string representation of a nomination"""
+        guild = self.bot.get_guild(Guild.id)
+
+        actor_id = nomination_object["actor"]
+        actor = guild.get_member(actor_id)
+
+        active = nomination_object["active"]
+        log.debug(active)
+        log.debug(type(nomination_object["inserted_at"]))
+
+        start_date = self._get_human_readable(nomination_object["inserted_at"])
+        if active:
+            lines = textwrap.dedent(
+                f"""
+                ===============
+                Status: **Active**
+                Date: {start_date}
+                Actor: {actor.mention if actor else actor_id}
+                Reason: {nomination_object["reason"]}
+                Nomination ID: `{nomination_object["id"]}`
+                ===============
+                """
```

Comment:
Does this cause issues with leading and trailing newlines? Other instances that use triple quotes as parameters have done it a bit differently:

https://github.com/python-discord/bot/blob/django/bot/cogs/moderation.py#L157

Reply:
The difference between the two is not the newlines (there's still a newline after the `"""` in both cases, but the `strip` takes care of it later), but rather that in this case, the string starts the line after the opening `(`, indented like a list of arguments would if the `()` split over multiple lines.

In my eyes, doing this:
```py
textwrap.dedent("""
    hello
    this is my multiline comment
""")
```
looks like:
```py
some_function(arg1,
    arg2,
    arg3,
arg4)
```
Since part of the string definition is indented, part of it follows the opening `(` and part of it is squeezed before the closing `)` unindented (the closing quotes).

Reply:
Missed the strip afterwards. This is fine as-is.